config/software/ruby.rb

```diff
@@ -136,6 +136,10 @@
 
   if solaris_10? && version.satisfies?(">= 2.1")
     patch source: "ruby-no-stack-protector.patch", plevel: 1, env: patch_env
+    if version.satisfies?("= 2.4.4") && sparc?
```

Comment:
Version constraint necessary?

Reply:
Likely as it's patching the configure script.